variable/test/math_test.cpp

```diff
@@ -430,3 +430,21 @@ TEST(Variable, log10_out_arg) {
 TEST(Variable, log10_bad_unit) {
   EXPECT_THROW_DISCARD(log10(1.0 * units::s), except::UnitError);
 }
+
+TEST(Variable, rint_round) {
+  auto preRoundedVar = makeVariable<double>(Values{1.2, 2.9, 1.5, 2.5});
+  auto roundedVar = makeVariable<double>(Values{1, 3, 2, 2});
+  EXPECT_EQ(rint(preRoundedVar), roundedVar);
```

Comment:
Minor: You could potentially parameterise these tests on a template type argument to check for both `double` and `float` given they are the only supported types.

Reply:
I would recommend to not use templated test: We know that `transform` can handle this. In fact I think there entirety of these Variable-based tests should probably be removed, since the functions are auto-generated.

Reply:
I disagree with removing the Variable based tests as whilst they are auto generated, it will point out any potential flaws for specific use cases in the auto generated code.

Reply:
Can you give an example of what aspect may lack testing? If so, add it to `dataset/test/generated_test.cpp` please.

Reply:
I can't give a specific example at the moment, but I can foresee it being a potential problem. Sure we could remove these tests... but on the other hand there is no downside to keeping these besides minimal maintenance. The tests do ensure that `rint`, `ceil`, and `floor` are actually still generated for the C++ math.h so at a minimum we should keep that.